app/routes.js

```diff
@@ -88,12 +98,33 @@ router.post('/dwp_passported', function (req, res, next) {
 
 router.get('/dwp_passported', function (req, res) {
   _.set(req.session.data, 'means_assessment.benefits_status.passported', true);
-  res.render('dwp_passported');
+
+  let mvp = req.session.mvp;
+  let outOfScope = false;
+  if (mvp) {
+    let nino = _.get(req.session.data, 'client_details.client.national_insurance_number');
+    let partner = _.get(req.session.data, 'client_details.partner.first_name');
+    let hasPartner = !_.isEmpty(partner);
+    if (_.isEmpty(nino) || hasPartner) {
```

Comment:
I'm assuming `if (_.isEmpty(nino) || _.isEmpty(partner))` didn't work?

Reply:
All the double negatives got a bit confusing is all

Reply:
The condition is checking if partner is NOT empty (ie if they have a partner, they slip out of MVP scope). I could use !_.isEmpty(partner) but my thinking was that `hasPartner` would be more explicit and easier to read.